test/e2e/elasticsearch_test.go

```diff
@@ -236,7 +237,7 @@ func turnOnEsIndexCleaner(name string, exampleJaeger *v1.Jaeger) {
 	err = fw.Client.Update(context.Background(), exampleJaeger)
 	require.NoError(t, err)
 
-	err = WaitForCronJob(t, fw.KubeClient, namespace, fmt.Sprintf("%s-es-index-cleaner", name), retryInterval, timeout)
+	err = WaitForCronJob(t, fw.KubeClient, namespace, fmt.Sprintf("%s-es-index-cleaner", name), retryInterval, 3*time.Minute)
```

Comment:
As `timeout` is configurable, wondering if this needs to be an increment on that default? So if someone wanted to update the default, it also modifies the relative value for this timeout? e.g. `timeout + 1*time.Minute`